_src/Evasions/techniques/AVBypass.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title:  "Evasions: Bypassing Security Solutions"
```

Comment:
Imo, there is no necessity in this additional chapter - existing ones can be redacted to include the described techniques.